src/Utilities.UnitTests/AdditionalFileProviderTests.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+using Xunit;
+
+namespace Analyzers.Utilities.UnitTests
+{
+
```

Comment:
```suggestion
```

Reply:
I would avoid adding comments for basic code style issues like this. If the style matters as a policy, it will be enforced during the build via analyzers.

(Comments are allowed for cases where analyzers to support the policy do not exist, but if the analyzer exists and the repository chooses not to use it then it's a sign that the style does not constitute a policy that contributors are required to adhere to.)

Reply:
➡️ I'm going to resolve all comments that fall under this bucket to reduce noise in the pull request.